docs/getstarted.md

```diff
@@ -173,25 +173,22 @@ target = "binary"
 # Creating GraphDataset objects
 dataset_train = GraphDataset(
     hdf5_path = hdf5_paths,
-    subset = train_ids, 
+    subset = train_ids,
```

Comment:
Please add in this file as well (docs/getstarted.md) `train_ids = [<ids>]` and the other ids def as you did in the README :)